autoload/test/strategy.vim

```diff
@@ -115,12 +115,14 @@ function! test#strategy#neovim_sticky(cmd) abort
     endif
   endif
 
-  call chansend(l:buffers[0].variables.terminal_job_id, l:cmd)
   let l:win = win_findbuf(l:buffers[0].bufnr)
   if !len(l:win) && get(g:, 'test#neovim_sticky#reopen_window', 0)
     call s:neovim_reopen_term(l:buffers[0].bufnr)
   endif
 
+  call chansend(l:buffers[0].variables.terminal_job_id, l:cmd)
+
+  let l:win = win_findbuf(l:buffers[0].bufnr)
```

Comment:
Calling `win_findbuf` again only makes sense if you previously called `s:neovim_reopen_term`, how about making that function return the new `l:win`? You can easily do it by calling `win_getid()` between `sbuffer` and `win_gotoid()`:

```viml
  execute term_position . ' sbuffer ' . a:bufnr
  let l:new_window = win_getid()
  call win_gotoid(l:current_window)
  return l:new_window

... 

  let l:win = [s:neovim_reopen_term(l:buffers[0].bufnr)]
```